client/my-sites/plan-price/README.md

```diff
@@ -1,14 +1,56 @@
 # PlanPrice Component
 
-PlanPrice component is a React component used to display plan's price with a currency and a discount, if any.
-It can be used anywhere where a plan's price is required.
+The PlanPrice component is a React component used to display a plan's price with a currency and a discount, if any. It can be used anywhere where a plan's price is required.
 
 If you want to emphasize that a plan's price is discounted, use two `<PlanPrice>` components as below and wrap them in a
 flexbox container.
 
+## Usage
+
+PlanPrice can take a `productDisplayPrice` or a `rawPrice`  (deprecated) prop, though `productDisplayPrice` is the preferred option.
+
+A `productDisplayPrice` can be passed from the `/purchases` or `/plans` REST endpoints to a Redux store, for example:
+
+```jsx
+	import { connect } from 'react-redux';
+
+	const { productDisplayPrice } = purchase;
+
+	export default connect( ( state, props ) => {
+		const purchase = getByPurchaseId( state, props.purchaseId );
+		...more props
```

Comment:
This always bites me but for some reason our linter scans code in Markdown files, so this line and the "...more props" line down below will both cause errors.

You could fix this by just removing both `...more props` lines so it's valid JS.

Reply:
😬  There's still some errors here:
<img width="352" alt="Screen Shot 2022-05-26 at 6 39 22 PM" src="https://user-images.githubusercontent.com/2036909/170591054-6c21e038-2772-4c8e-a729-2aa6c4ea07d0.png">

What I find helpful is to take the code, paste it into an actual JS file, format it, then copy and paste it back into the Markdown file.